src/win/fs.c

```diff
@@ -70,10 +70,7 @@
 #define SET_REQ_RESULT(req, result_value)                                   \
   do {                                                                      \
     req->result = (result_value);                                           \
-    if (req->result == -1) {                                                \
-      req->sys_errno_ = _doserrno;                                          \
-      req->result = uv_translate_sys_error(req->sys_errno_);                \
-    }                                                                       \
+    assert(req->result != -1);                                              \
```

Comment:
Hm, at least `fs__mktemp()` appears to be calling `SET_REQ_RESULT(req, -1)`...

Reply:
Yeah, there were a couple places. That one happened to be calling the wrong function before though anyways (mkstemp was not setting _doserrno either. It just happened to become a test failure when mkdtemp also stopped setting _doserrno)